test/Unit/ContentProviderTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace WMDE\Fundraising\HtmlFilter\Test\Unit;
+
+use Twig_Environment;
+use WMDE\Fundraising\HtmlFilter\ContentProvider;
+use ReflectionClass;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Poor man's unit tests in the absence of clean DI on ContentProvider
+ *
+ * Class ContentProviderTest
+ * @package WMDE\Fundraising\HtmlFilter\Test\Unit
```

Comment:
The `Class ContentProviderTest` line can be deleted.
We're not doing `@package` tags at the moment.

Please add `@covers WMDE\Fundraising\HtmlFilter\ContentProvider`